.circleci/config.yml

```diff
@@ -7,7 +7,12 @@ jobs:
       - checkout
       - run: docker build -t protoc-gen-validate .
       - run: docker run --rm protoc-gen-validate ci
-  
+      - run:
+          command: | # Upload if on master branch
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
is a no-op if package by the same version as in setup.cfg already exists. 

TODO: pass envoyproxy's PyPI login. Asking Matt Klein to create an account.